apps/web/modules/ui/components/date-picker/index.tsx

```diff
@@ -5,7 +5,7 @@ import { Button } from "@/modules/ui/components/button";
 import { Popover, PopoverContent, PopoverTrigger } from "@/modules/ui/components/popover";
```

Comment:
This same component appears different in the survey editor and personal links.

Survey editor:
![Image](https://github.com/user-attachments/assets/f8d2a5eb-1de7-4fb3-9947-04cbc314691d)

Personal links:
![Image](https://github.com/user-attachments/assets/6125c6de-907e-445d-a9eb-dc571757fcbd)